src/mods/common/analyzer/append/mathutils.mod.rst

```diff
@@ -268,12 +411,43 @@
 
       :type key: int
       :rtype: float
+      :option function: overload
+
+   .. method:: __getitem__(key)
+
+      :type key: slice
+      :mod-option arg key: skip-refine
+      :rtype: tuple[float, ...]
+      :mod-option rtype: skip-refine
+      :option function: overload
+
+   .. method:: __getitem__(key)
+
+      :type key: int | slice
+      :mod-option arg key: skip-refine
+      :rtype: float | tuple[float, ...]
+      :mod-option rtype: skip-refine
 
    .. method:: __setitem__(key, value)
 
       :type key: int
       :type value: float
-      :rtype: float
+      :option function: overload
+
+   .. method:: __setitem__(key, value)
+
+      :type key: slice
+      :mod-option arg key: skip-refine
+      :type value: collections.abc.Iterable[float]
+      :mod-option arg value: skip-refine
+      :option function: overload
+
+   .. method:: __setitem__(key, value)
+
+      :type key: int | slice
+      :mod-option arg key: skip-refine
+      :type value: float | collections.abc.Iterable[float]
+      :mod-option arg value: skip-refine
```

Comment:
Could you explain why this function with the union data type for `key` and `value` is needed?
This is same as others.

Reply:
I don't know, I've just been replicating the first mod file with `@overload`, which does the same thing:
https://github.com/nutti/fake-bpy-module/blob/d3672f21378b188f5fbcb2321e5659783000e337/src/mods/common/analyzer/append/mathutils.mod.rst?plain=1#L111-L126

Reply:
Research reveals that the non-`@overload`-decorated function is the [runtime implementation](https://docs.python.org/3/library/typing.html#typing.overload), which [should not be included in type stubs](https://typing.readthedocs.io/en/latest/source/stubs.html#functions-and-methods).

I shall remove them from this PR, and submit another to remove the existing ones.